src/spring/HISTORY.md

```diff
@@ -1,13 +1,11 @@
 Release History
 ===============
 
-1.1.7
+1.1.8
```

Comment:
Why not use version `1.1.7` but skip it?

Reply:
Oh, got catch! Just released that feature owner updated `1.1.7` in history but not `setup.py`. Yes, we can use `1.1.7`. Updated!